config.fullstack.test.yaml

```diff
@@ -135,6 +135,32 @@ wiktionary_project: &wiktionary_project
                 options: *proxy_options
           /{api:sys}: *default_sys
 
+commons_project: &commons_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - path: projects/v1/default.wmf.yaml
+                options: *default_options
+              - path: v1/recommend-caption.yaml
+                options: '{{options.recommendation}}'
+              - path: projects/proxy.yaml
+                options: *proxy_options
+          /{api:sys}: *default_sys
```

Comment:
This portion actually needs to be transferred to the production config. Want me to do it or want me to show you how it's done @mateusbs17 ?

Reply:
Also, we need to update `config.fronted.wikimedia.yaml` similarly.

Reply:
Went ahead and made a prod config change for you https://gerrit.wikimedia.org/r/c/mediawiki/services/restbase/deploy/+/519500

Reply:
Same question.